screens/HomeScreenLayout.tsx

```diff
@@ -82,15 +82,10 @@ export const HomeScreenLayout: React.FC<RootRouteProps> = props => {
   const [isRTL] = useState(I18nManager.isRTL);
 
   var HomeScreenOptions = {
-    headerLeft: () =>
-      isIOS()
-        ? SvgImage.InjiLogo()
-        : isRTL
-        ? screenOptions
-        : SvgImage.InjiLogo(),
+    headerLeft: () => (isIOS() || !isRTL ? SvgImage.InjiLogo() : screenOptions),
```

Comment:
Please add the  if condition in a parantheses to avoid confusion:
(isIOS() || !isRTL) ? SvgImage.InjiLogo() : screenOptions
Do the same at both the place

Reply:
Done